mediathread/templates/navbar.html

```diff
@@ -1,90 +1,119 @@
-                        Manage Course
-                    </a>
-                </li>
-            {% endif %}
+<a title="Mediathread" class="navbar-brand" href="/">
+  <img src='{% static "img/logo-mediathread-bgdark.svg" %}' height="50" alt="Mediathread" />
+</a>
+<button class="navbar-toggler" type="button" data-toggle="collapse"
+        data-target="#navbarNavDropdown" aria-controls="navbarNavDropdown"
+        aria-expanded="false" aria-label="Toggle navigation"><span class="navbar-toggler-icon"></span></button>
+<div class="collapse navbar-collapse" id="navbarNavDropdown">
+  <ul class="navbar-nav w-100">
 
-            {% if user.is_impersonate %}
-                <li class="nav-item">
-                    <a class="btn btn-outline-danger" role="button"
-                       href="/impersonate/stop/" title="Impersonate Stop">
-                        Impersonate Stop
-                    </a>
-                </li>
-            {% endif %}
+    <!-- Only show if not logged in -->
+    {% if not request.user.is_authenticated %}
+        <li class="nav-item dropdown">
+        <a class="nav-link dropdown-toggle" href="#"
+               id="navbarDropdown" role="button"
+               data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                Learn More
+        </a>
+        <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+            <a class="dropdown-item" href="/about">About Mediathread</a>
+            <a class="dropdown-item" href="/testimonials">Testimonials</a>
+            <a class="dropdown-item" href="/history">Mediathread History</a>
+        </div>
+        </li>
+    {% endif %}
+
+        <li class="nav-item dropdown">
+        <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Use Mediathread </a>
+        <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+            <a class="dropdown-item" href="value.html">Pedagogical Value</a>
+            <a class="dropdown-item" href="examples.html">Examples</a>
+        </div>
+        </li>
+        <li class="nav-item dropdown">
+        <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"> Get Support </a>
+        <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+            <a class="dropdown-item" href="http://support.ctl.columbia.edu/knowledgebase/topics/6593-mediathread">Faculty</a>
+            <a class="dropdown-item" href="http://support.ctl.columbia.edu/knowledgebase/topics/6593-mediathread">Students</a>
+            <a class="dropdown-item" href="{% url 'contact-us' %}">Contact Us</a>
+        </div>
+        </li>
+
+    <!-- Only show if not logged in -->
+    {% if not request.user.is_authenticated %}
+        <li class="nav-item"> <a class="nav-link" href="getinvolved.html"> Get Involved </a> </li>
```

Comment:
Update this link to a url

Reply:
Ah yes. I redid the links in my new branch. Forgot I could just do it here. :D